ldi-core/ldes-client/tree-node-supplier/src/test/resources/features/member-supplier.feature

```diff
@@ -54,3 +54,40 @@ Feature: MemberSupplier
       | SQLITE                   |
       | FILE                     |
       | POSTGRES                 |
+
+  Scenario Outline: Obtaining the members from multiple endpoints
+    Given Starting urls
```

Comment:
I know that it is a bit more work but it is crucial for the way everything works. Can you add a second fragment page to group 1 and verify that those members are processed after the first page of group 2 is processed? To verify the acceptance criteria that the groups should be processed 'parallel'